aquila/urls.py

```diff
@@ -29,4 +31,8 @@
     path('groupings/<int:pk>/edit/', GroupingUpdateView.as_view(), name="groupings-update"),
```

Comment:
```suggestion
    path('groupings/<int:pk>/update/', GroupingUpdateView.as_view(), name="groupings-update"),
```
Aligning naming conventions